### ISSUE TITLE
Helm: Add support for custom args

### ISSUE THREAD

**Issue by Brice187** [burningalchemist/sql_exporter#605]
# Problem: 

I am using the sql_exporter helmchart in a gitops environment. 

Developing custom crafted metrics is currently toilsome because getting a fresh developed collector into my dev environment, i have to commit/push the change to the feature-branch, sync the gitops repo, wait for redeployment of the sql_exporter pod and than i have to set a new port-forward to my localhost to curl the metrics endpoint.

## Solution:

My proposed solution would be:

```
kubectl exec -it deployment/sql-exporter -- sh

vi /tmp/mymetric.dev.yaml
```

the sql_exporter.yml would glob the collectors `/tmp/*.dev.yam` and after every change I would get feedback when I quickly reload the config.

## long story short

There are three possible solutions

1. template container command args

The args of the container are hardcoded: https://github.com/burningalchemist/sql_exporter/blob/6e4d3df8421a84f3efad2158ae98b5bc3e42d8b2/helm/templates/deployment.yaml#L56

It would be nice if I could add `-web.enable-reload` via helm.

Bonus: 

```yaml
reload:
  enabled: true # (default false)
```
^ Would do the job to ;)

2. Create ENV var to enable reload

3. Default to web.enable-reload
